pkg/api/validation/validation.go

```diff
@@ -1602,6 +1602,10 @@ func ValidatePersistentVolumeClaimStatusUpdate(newPvc, oldPvc *api.PersistentVol
 	if len(newPvc.Spec.AccessModes) == 0 {
 		allErrs = append(allErrs, field.Required(field.NewPath("Spec", "accessModes"), ""))
 	}
+	if !utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) && len(newPvc.Status.Conditions) > 0 {
```

Comment:
I don't think your feature gate has to protect conditions.  Conditions are something any controller can set.  You can remove this check.

Reply:
Keep the test if it's valid though.

Reply:
the conditions field is new to this feature... it's a tad odd to add it and unconditionally enable it with only one current use

Reply:
yeah, I have left this check in place for now.